### PR TITLE
Importer supports local authority folders

### PR DESCRIPTION
**Pull request by notapatch**

### CODE REVIEW

File: app/importers/planning_applications_importer.rb [hunk 1]
```diff
@@ -5,26 +5,35 @@ def initialize(local_authority_name:)
...
- def call
-   import_planning_applications
- rescue StandardError => exception
-   log_exception(exception)
- end
+  def call
+    import_planning_applications
+  rescue StandardError => exception
+    log_exception(exception)
+  end
```
Comment (allbecauseyoutoldmeso) [inline app/importers/planning_applications_importer.rb:12]: It looks like github actions is configured to run rubocop, but shouldn't it have picked up on indentation issues?

**Review by allbecauseyoutoldmeso — APPROVED** [review]
👍

File: app/importers/planning_applications_importer.rb (hunk 1 above)
Comment (EGiataganas) [reply app/importers/planning_applications_importer.rb:12]: Agreed, 👍🏻. I think we should make Rubocop more strict in order to pick it

File: app/importers/planning_applications_importer.rb [hunk 2]
```diff
@@ -5,26 +5,35 @@ def initialize(local_authority_name:)
...
 
- private
+  private
```
Comment (EGiataganas) [inline app/importers/planning_applications_importer.rb:14]: Spacing

File: app/importers/planning_applications_importer.rb (hunk 1 above)
Comment (notapatch) [reply app/importers/planning_applications_importer.rb:12]: There's virtually no rules. It's programming in madmax world. And the application I'm using is adding in extra space here and there.

**Review by EGiataganas — APPROVED** [review]

**Pull request merged**